docs/standard/components.md

```diff
@@ -1,80 +1,88 @@
+- Optionally, development tools. Some development tools are shared among multiple implementations.
 
-There are 3 primary .NET runtimes which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
+There are four primary .NET implementations that Microsoft actively develops and maintains: .NET Core, .NET Framework, Mono, and UWP.
 
 ### .NET Core
 
-.NET Core is a cross-platform runtime optimized for server workloads.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on .NET Core.  It is the runtime used by ASP.NET Core and the Universal Windows Platform (UWP).  It is modern, efficient, and designed to handle server and cloud workloads at scale.
+.NET Core is a cross-platform implementation of .NET optimized for server workloads. It's modern, efficient, and designed to handle server and cloud workloads at scale. It implements the .NET Standard, so code that targets the .NET Standard can run on .NET Core. ASP.NET Core runs on .NET Core.
 
 To learn more about .NET Core, see the [.NET Core Guide](../core/index.md).
 
 ### .NET Framework
 
-.NET Framework is the historical .NET runtime that has existed since 2002.  It is the same .NET Framework existing .NET developers have always used.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on the .NET Framework.  It contains additional Windows-specific APIs, such as APIs for Windows desktop development with Windows Forms and WPF.  .NET Framework is optimized for building Windows desktop applications.
+The.NET Framework is the original .NET implementation that has existed since 2002. It's the same .NET Framework that existing .NET developers have always used. Versions 4.5 and later implement the .NET Standard, so code that targets the .NET Standard can run on those versions of the .NET Framework. It contains additional Windows-specific APIs, such as APIs for Windows desktop development with Windows Forms and WPF. The .NET Framework is optimized for building Windows desktop applications.
 
 To learn more about the .NET Framework, see the [.NET Framework Guide](../framework/index.md).
 
-### Mono for Xamarin
+### Mono
 
-Mono is the runtime used by Xamarin apps.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on Xamarin apps.  It contains additional APIs for iOS, Android, Xamarin.Forms, and Xamarin.Mac.  It is optimized for building mobile applications on iOS and Android.
+Mono is the .NET implementation used by Xamarin apps. It implements the .NET Standard, so code that targets the .NET Standard can run on Xamarin apps. It contains additional APIs for iOS, Android, Xamarin.Forms, and Xamarin.Mac. It's optimized for building mobile applications on iOS and Android.
```

Comment:
The Xamarin.iOS/Xamarin.Android/Xamarin.Mac APIs are not part of Mono. I think that by the distinction that's being used here, Xamarin.iOS, Xamarin.Android and Xamarin.Mac are all ".NET implementations" that use the Mono runtime.

Xamarin.Forms isn't part of any of these - it's a NuGet that runs on UWP as well as Xamarin.iOS and Xamarin.Android.

Additionally, Mono has a desktop/server ".NET implementation" that is mostly compatible with .NET Framework, modulo Windows-specific APIs and a few other things. It doesn't have a formal name and is usually called just "Mono" or "Mono desktop framework". This implementation is what VS for Mac runs on.